infrastructure/ai/providers/openai_provider.py

```
@@ -61,11 +73,13 @@ async def generate(
                 except (openai.NotFoundError, openai.BadRequestError, RuntimeError) as e:
                     logger.info(f"Responses API unsupported for {base_url}, falling back to chat completions: {str(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)